Zend/zend_vm_def.h

```diff
@@ -457,6 +457,20 @@ ZEND_VM_COLD_CONSTCONST_HANDLER(16, ZEND_IS_IDENTICAL, CONST|TMP|VAR|CV, CONST|T
 	ZEND_VM_SMART_BRANCH(result, 1);
 }
 
+ZEND_VM_HANDLER(196, ZEND_CASE_STRICT, TMP|VAR, CONST|TMP|VAR|CV)
```

Comment:
Why aren't there handler implementations for `CONST` or `CV` for op1?

E.g. what will happen if opcache optimizations (opcache.optimization_level=0) are turned off for `function test(int $x, int $y) { match ($x) { $y => print("same\n"); }; }`, or `function test(int $x, int $y) { match (2) { $y => print("two\n"); }; }`

Otherwise, I didn't have any comments

Reply:
The reason is that `CONST` and `CV` are handled by `IS_IDENTICAL`.

See https://github.com/php/php-src/blob/c37f0385f6553168ed40e9cbfe59ca6efbdf7020/Zend/zend_compile.c#L5267